azure-pipelines.yml

```
@@ -459,6 +460,14 @@ jobs:
       parallel: true
       parallelCount: 8
 
+  - task: PowerShell@2
+    displayName: 'Publish Documentation HTML'
+    inputs:
+      targetType: 'filePath'
+      filePath: '$(Build.SourcesDirectory)\doxygen\PublishDocs.ps1'
+    condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'))
```

Thinking about this a little more. Maybe we could add

```suggestion
    condition: and(eq(variables['System.CollectionId'], 'cb55739e-4afe-46a3-970f-1b49d8ee7564'), and(succeeded(), ne(variables['Build.Reason'], 'PullRequest')))
```

Though its super ugly